content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -461,15 +461,17 @@ The method calls above would need to be put inside a `script` directive:
 
 pipeline {
     agent none
-    stage ('Example') {
-        steps {
-            // log.info 'Starting' // <1>
-            script { // <2>
-                log.info 'Starting'
-                log.warning 'Nothing to do!'
+    stages{ 
+        stage ('Example') {
+            steps {
+                // log.info 'Starting' // <1>
+                script { // <2>
+                    log.info 'Starting'
+                    log.warning 'Nothing to do!'
+                }
             }
         }
-    }
+    }    
```

Comment:
I would prefer to not have trailing white space on the end of a line, but that should not block this pull request.
```suggestion
    }
```